### ISSUE TITLE
### <span aria-hidden="true">❌</span> Deploy Preview for *stackblitz-docs* failed. **[Why did it fail? →](https://app.netlify.com/projects/stackblitz-docs/deploys/68bf644c43edcf0008a2e444?deploy_diagnostics=true)**

### ISSUE THREAD

**Issue by Jobayer071** [stackblitz/docs#246]
### <span aria-hidden="true">❌</span> Deploy Preview for *stackblitz-docs* failed. **[Why did it fail? →](https://app.netlify.com/projects/stackblitz-docs/deploys/68bf644c43edcf0008a2e444?deploy_diagnostics=true)**

|  Name | Link |
|:-:|------------------------|
|<span aria-hidden="true">🔨</span> Latest commit | 8c909dd4c5be3d639955326a3c33cc79701bd3e9 |
|<span aria-hidden="true">🔍</span> Latest deploy log | https://app.netlify.com/projects/stackblitz-docs/deploys/68bf644c43edcf0008a2e444 |

_Originally posted by @netlify[bot] in https://github.com/stackblitz/docs/issues/244#issuecomment-3268325144_

**Comment by Jobayer071** [comment]
> ### ❌ Deploy Preview for _stackblitz-docs_ failed. **[Why did it fail? →](https://app.netlify.com/projects/stackblitz-docs/deploys/68bf644c43edcf0008a2e444?deploy_diagnostics=true)**
> Name	Link
> 🔨 Latest commit	[8c909dd](https://github.com/stackblitz/docs/commit/8c909dd4c5be3d639955326a3c33cc79701bd3e9)
> 🔍 Latest deploy log	https://app.netlify.com/projects/stackblitz-docs/deploys/68bf644c43edcf0008a2e444
> _Originally posted by [@netlify](https://github.com/netlify)[bot] in [#244 (comment)](https://github.com/stackblitz/docs/pull/244#issuecomment-3268325144)_
